### PR TITLE
Fixes #3505 - Fixed Example 15

### LINKED ISSUE #3505: Example 15 is incorrect
On example 15 Enable-WSManCredSS needs to use a role, also will be nice to know:

- the remote machine needs the server role, 
- the local machine has to run winrm server also, which is not needed for the most of the other examples

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9b35c00a-bb1b-f58b-6070-1e777c3cfc25
* Version Independent ID: 05657c1c-5390-4a39-53d6-9a34551670e9
* Content: [Invoke-Command (Microsoft.PowerShell.Core)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/invoke-command?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Core/Invoke-Command.md](https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/Invoke-Command.md)
* Product: **powershell**
* GitHub Login: @SteveL-MSFT
* Microsoft Alias: **slee**

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

- Fixes [AB#1611400](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1611400)
- Fixes #3505
- Copyedits, style, paragraph reflows
- Changed Examples 6, 14, 15, and 16 to use splatted parameters

Scores
Original version: 90
Updated version: 94

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md [hunk 1]
```diff
@@ -1151,49 +1173,61 @@ returns the output of the invoked command, which is the value of the **ScriptBlo
...
-  indicates that you cannot connect to the PSSession because it is connected to another session.
-
-  For more information about the values of the **State** property of sessions, see
-  [RunspaceState Enumeration](https://msdn.microsoft.com/library/system.management.automation.runspaces.runspacestate).
-
-  For more information about the values of the **Availability** property of sessions, see
-  [RunspaceAvailability Enumeration](https://msdn.microsoft.com/library/system.management.automation.runspaces.runspaceavailability).
+On Windows Vista, and later versions of the Windows operating system, to use the **ComputerName**
+parameter of `Invoke-Command` to run a command on the local computer, you must open PowerShell by
+using the Run as administrator option.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Change to:

you must run PowerShell using the **Run as administrator** option.

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md [hunk 2]
```diff
@@ -1151,49 +1173,61 @@ returns the output of the invoked command, which is the value of the **ScriptBlo
...
+
+`Set-Item -Path WSMan:\Localhost\Client\TrustedHosts -Value \<Remote-Computer-Name\>`
+
+In PowerShell 2.0, you can't use the `Select-Object` cmdlet to select the **PSComputerName**
+property of the object that `Invoke-Command` returns. Instead, to display the value of the
+**PSComputerName** property, use the dot method to get the **PSComputerName** property value
+($result.PSComputerName), use a **Format** cmdlet, such as the `Format-Table` cmdlet, to display the
+value of the **PSComputerName** property, or use a `Select-Object` command where the value of the
+property parameter is a calculated property that has a label other than PSComputerName. This
+limitation doesn't apply to PowerShell 3.0 or later versions.
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Strike this whole paragraph about PSv2.0.

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md [hunk 3]
```diff
@@ -1151,49 +1173,61 @@ returns the output of the invoked command, which is the value of the **ScriptBlo
...
+
+When you disconnect a **PSSession**, such as by using **InDisconnectedSession**, the session state
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Change to:

When you disconnect a **PSSession**, by using the **InDisconnectedSession** parameter, the session state

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md [hunk 4]
```diff
@@ -1151,49 +1173,61 @@ returns the output of the invoked command, which is the value of the **ScriptBlo
...
+is Disconnected and the availability is **None**. The value of the **State** property is relative to
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Bold "Diconnected"

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md (hunk 2 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Fixed

File: reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md (hunk 4 above)
Comment (davidsmatlak) [reply reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md]: Fixed

**Pull request merged**